andela/balder-ah-backend
# #162170701 Integrate HoundCI to validate the codebase with standard style guides

### What does this PR do?
- Automatically validate the codebase with standard style guides
### Description of Task to be completed
- Integrate HoundCI with badge attached to readme
- Create _*.hound.yml*_ file containing the style guides and rules
### How should this be manually tested?
- Clone this repo and cd into it
- Write your codebase and ensure no ESLINT rule is violated
- Push to the remote branch and raise a pull request
### Any background context you want to provide?
- None
### What are the relevant pivotal tracker stories?
- [#162170701](https://www.pivotaltracker.com/story/show/162170701)

### If No Style Guide is Violated (screenshot)

![hound-pass](https://user-images.githubusercontent.com/25563661/49078632-d3328e00-f23e-11e8-9179-cc41e73c0352.png)
